### PR TITLE
Add additional context to view check hook output

### PR DESCRIPTION
**Pull request by rgeniesse**
## Description
Adds more details on where to see the output of a check `hook` command.

## Motivation and Context
In working with `hooks` for the first time, I found it difficult to find where you can view the output. Until https://github.com/sensu/web/issues/38 is resolved as it is a more natural place to find this data, users should be provided more details on how to do so.

## Review Instructions
- [x] Copy to all versions

### CODE REVIEW

**Review by apaskulin — COMMENTED** [review]
Thanks for adding this content! I think this will be a super helpful example. I do think this might be more readable and compact in YML instead of JSON.

File: content/sensu-go/5.9/reference/hooks.md [hunk 1]
```diff
@@ -42,6 +42,34 @@ See the [check specification][6] to configure the `check_hooks` attribute.
...
+{{< /highlight >}}
+
+{{< highlight shell >}}
+type: Event
+api_version: core/v2
+metadata:
+  namespace: default
+spec:
+  check:
+    "...": "..."
```
Comment (apaskulin) [inline content/sensu-go/5.9/reference/hooks.md]:
```suggestion
    ...
```

**Review by apaskulin — APPROVED** [review]

File: content/sensu-go/5.9/reference/hooks.md (hunk 1 above)
Comment (rgeniesse) [reply content/sensu-go/5.9/reference/hooks.md]: Oh weird, I guess the converter I used doesn't know what to do with that.

**Pull request merged**